### DISCUSSION TITLE
Add support for geometric mean and harmonic mean

### DISCUSSION THREAD

**Discussion by lulunac27a** [dathere/qsv#2188] [Ideas]
I would like to calculate the geometric mean and harmonic mean of the array of different numbers.
The geometric mean is the nth root of the product of each number of the array and the harmonic mean is the sum of reciprocal for each number in the array multiplied by the array length.
Sources:
https://byjus.com/maths/geometric-mean/
https://byjus.com/maths/harmonic-mean/

**Comment by jqnatividad** [comment]
Adding to the backlog.
